radar-upload-frontend/src/components/Patient/PatientTable/index.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <v-data-table
+    :items="items"
+    single-expand
+    :search="searchText"
+    :expanded.sync="expandedItems"
+    show-expand
+    :headers="headers"
+    hide-default-header
+    item-key="sequence"
+    :loading="loading"
+    loading-text="Downloading patients info, please wait"
+    @item-expanded="getPatientRecords"
+  >
+    <template #item.updatedAt="{item}">
+      <td class="pl-0">
+        {{ item.updatedAt | localTime }}
+      </td>
+    </template>
+
+    <template #expanded-item="{item}">
+      <td
+        :colspan="$vuetify.breakpoint.name==='xs' ? 0 :12"
+        class="pa-2"
+      >
+        <Records
+          :patient-records="patientRecords"
+          :loading="fileLoading"
+          :error="fileLoadingError"
+        >
+          <template #fileListSubHeader>
+            <UploadButton
+              :upload-info="{
+                userId: item.patientId,
+                projectId: currentProject
+              }"
+              @addUploadingFile="addUploadingFile"
+              @startUploading="startUploading"
+              @uploadFailed="uploadFailed"
+            />
+          </template>
+        </Records>
+      </td>
+    </template>
+  </v-data-table>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import UploadButton from './UploadButton';
+import Records from './Records';
+import patientAPI from '@/axios/patient';
+import fileAPI from '@/axios/file';
+
+export default {
+  components: {
+    UploadButton,
+    Records,
+  },
+  props: {
+    isActive: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      loading: false,
+      fileLoading: false,
+      fileLoadingError: '',
+      expandedItems: [],
+      items: [
+      ],
+      patientRecords: [
+      ],
+      headers: [
+        { text: '#', value: 'sequence' },
+        { text: 'Patient name', value: 'patientName' },
```

Comment:
We do not have `patient name`. Clinicians might think this is the actual patient name, where as we do not store any personal data like that. I would just called it `external-identifier` or `readable-identifier`. We can call it something else, but maybe not `patient name`, IMO. Plus, could you please also add the user `id` (the UUID). This is what attached to the records. If someone wants to filter the records based on `User`, they cannot find the right information on `patients` view

Reply:
As above, wording could be changed if confirmed.

Reply:
What does "officially confirmed" mean?

Reply:
It's called "External ID" in ManagementPortal, let's use that here as well.